src/main/java/com/notably/logic/commands/NewCommand.java

```diff
@@ -27,6 +31,7 @@ public NewCommand(Block block) {
      */
     public void execute(Model notablyModel) throws CommandException {
         requireNonNull(notablyModel);
+        logger.info("Executing NewCommand");
         try {
             notablyModel.addBlockToCurrentPath(toAdd);
         } catch (DuplicateBlockException ex) {
```

Comment:
Let's add another `logger.info()` after the `try-catch` execution? Just to be consistent with the other commands' logging